doc/api/path.markdown

```diff
@@ -117,6 +119,20 @@ path.format({
 '/file.txt'
 ```
 
+An example on Windows:
+
+```js
+path.format({
+    root : "C:\\",
+    dir : "C:\\path\\dir",
+    base : "file.txt",
+    ext : ".txt",
+    name : "file"
+})
+// returns
+'C:\\path\\dir\\file.txt'
```

Comment:
This line should probably also be commented or moved to the end of the previous line.